app/src/main/java/com/example/androidapplicationtask/AyaatRecyclerAdapter.kt

```diff
@@ -2,61 +2,52 @@ package com.example.androidapplicationtask
 
 import android.annotation.SuppressLint
 import android.content.Context
+import android.renderscript.ScriptGroup.Binding
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.TextView
+import androidx.databinding.DataBindingUtil
 import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.Adapter
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+import com.example.androidapplicationtask.databinding.AyatItemViewBinding
 
 class AyaatRecyclerAdapter(
     val ayaat: List<AyaatData>,
-    val onItemClickListenerForAyaat: OnItemClickListenerForAyaat,
+    val listner: OnItemClickListenerForAyaat,
```

Comment:
give proper name for listener i.e 'OnItemClickListene'

Reply:
done